src/utils/token.js

```diff
@@ -0,0 +1,5 @@
+const AUTH_TOKEN = 'auth-token';
+
+export const getToken = () => localStorage.getItem(AUTH_TOKEN);
```

Comment:
These `localStorage` calls dovetail with a utility that I asked @jtan381 to add to #5. I think we should add an abstraction layer to access `localStorage` that calls `JSON.stringify` on set and `JSON.parse` on get. Once that PR gets merged, you can use it.

Reply:
Got it